### PR TITLE
[ENT-2082] Added Source to the enterprise enrollment background task

### PR DESCRIPTION
**Pull request by staubina**
**Merge checklist:**
- [x] Any new requirements are in the right place (do **not** manually modify the `requirements/*.txt` files)
    - `base.in` if needed in production but edx-platform doesn't install it
    - `test-master.in` if edx-platform pins it, with a matching version
    - `make upgrade && make requirements` have been run to regenerate requirements
- [x] `make static` has been run to update webpack bundling if any static content was updated
- [x] [Version](https://github.com/edx/edx-enterprise/blob/master/enterprise/__init__.py) bumped
- [x] [Changelog](https://github.com/edx/edx-enterprise/blob/master/CHANGELOG.rst) record added
- [x] Translations updated

**Post merge:**
- [ ] Tag pushed and a new [version](https://github.com/edx/edx-enterprise/releases) released
- [ ] Version pushed to [PyPI](https://pypi.org/project/edx-enterprise/)
- [ ] PR created in [edx-platform](https://github.com/edx/edx-platform) to upgrade dependencies (including edx-enterprise)

### CODE REVIEW

File: enterprise/tasks.py [hunk 1]
```diff
@@ -35,11 +35,13 @@ def create_enterprise_enrollment(course_id, enterprise_customer_user_id):
 
     enterprise_customer = enterprise_customer_user.enterprise_customer
     if enterprise_customer.catalog_contains_course(course_id):
+        LOGGER.info('***** TEST')
```
Comment (justinhynes) [inline enterprise/tasks.py]: Did you mean to leave this log message in?

File: enterprise/migrations/0081_UpdateEnterpriseEnrollmentSource.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('enterprise', '0079_AddEnterpriseEnrollmentSource'),
+        ('enterprise', '0080_auto_20191113_1708'),
+    ]
+
+    operations = [
+        migrations.RunPython(update_source, migrations.RunPython.noop)
```
Comment (crice100) [inline enterprise/migrations/0081_UpdateEnterpriseEnrollmentSource.py]: Should this have a rollback?

File: enterprise/tasks.py (hunk 1 above)
Comment (staubina) [reply enterprise/tasks.py]: No will be removed soon.

File: enterprise/migrations/0081_UpdateEnterpriseEnrollmentSource.py (hunk 2 above)
Comment (staubina) [reply enterprise/migrations/0081_UpdateEnterpriseEnrollmentSource.py]: Adjusted this.

**Pull request merged**